src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Shapes implements Area, Draw {
```

Comment:
Тобі не потрібно імплементувати інтерфейси які імплементовані в класі від якого ти наслідуєшся
```suggestion
public class Circle extends Shapes {
```